cookbook-react/src/recipes/screens/recoverPassword/cookbook.json

```diff
@@ -0,0 +1,10 @@
+{
+  "thumbnail": {
+    "type": "iframe",
+    "url": "https://cookbook-react.now.sh/?category=screens&component=recoverPassword&compact=true"
+  },
+  "detail": {
+    "type": "iframe",
+    "url": "https://cookbook-react.now.sh/?category=screens&component=recoverPassword&compact=true"
```

Comment:
we might think another way to achieve this, maybe having the route here, but where this is consumed use a environment variable for the url domain part, because if we migrate from now.sh the changes are big, am i wrong?

Reply:
Yep, we discussed that with @pabloferro, but we have to change some things in order for that to work. Right now this is a JSON file, it can't accept variables.